mbslave/replication.py

```
@@ -19,10 +19,22 @@
 from contextlib import ExitStack
 
 import prometheus_client
+from tqdm import tqdm
 
 logger = logging.getLogger(__name__)
 
 
+class TqdmFileReader(object):
```

Ruff (0.14.10)

Class `TqdmFileReader` inherits from `object`

Remove `object` inheritance

(UP004)